types/p5/src/image/loading_displaying.d.ts

```diff
@@ -5,144 +5,174 @@ import * as p5 from '../../index';
-         *   arguments will correspond to seconds, meaning that
-         *   3 seconds worth of animation will be created. If
-         *   'frames', the arguments now correspond to the
-         *   number of frames you want your animation to be, if
-         *   you are very sure of this number.
-         *
-         *   This may be called in setup, or, like in the
-         *   example below, inside an event function, like
-         *   keyPressed or mousePressed.
-         *   @param filename File name of your gif
-         *   @param duration Duration in seconds that you wish
-         *   to capture from your sketch
-         *   @param options An optional object that can contain
-         *   five more arguments: delay, specifying how much
-         *   time we should wait before recording; units, a
-         *   string that can be either 'seconds' or 'frames'.
-         *   By default it's 'seconds’; silent, a boolean that
+         *   Generates a gif from a sketch and saves it to a
+         *   file. saveGif() may be called in setup() or at any
+         *   point while a sketch is running. The first
+         *   parameter, fileName, sets the gif's file name. The
+         *   second parameter, duration, sets the gif's
+         *   duration in seconds.
+         *
+         *   The third parameter, options, is optional. If an
+         *   object is passed, saveGif() will use its
+         *   properties to customize the gif. saveGif()
+         *   recognizes the properties delay, units, silent,
+         *   notificationDuration, and notificationID.
+         *   @param filename file name of gif.
+         *   @param duration duration in seconds to capture
+         *   from the sketch.
+         *   @param [options] an object that can contain five
+         *   more properties: delay, a Number specifying how
+         *   much time to wait before recording; units, a
+         *   String that can be either 'seconds' or 'frames'.
+         *   By default it's 'seconds’; silent, a Boolean that
          *   defines presence of progress notifications. By
-         *   default it’s false; notificationDuration, a number
+         *   default it’s false; notificationDuration, a Number
          *   that defines how long in seconds the final
-         *   notification will live. 0, the default value,
-         *   means that the notification will never be removed;
-         *   notificationID, a string that specifies the
-         *   notification DOM element id. By default it’s
+         *   notification will live. By default it's 0, meaning
+         *   the notification will never be removed;
+         *   notificationID, a String that specifies the id of
+         *   the notification's DOM element. By default it’s
          *   'progressBar’.
          */
-        saveGif(filename: string, duration: number, options: object): void;
+        saveGif(filename: string, duration: number, options?: object): void;
```

Comment:
This is another object where I would love to see us define the `options` interface.